src/styles/blocks/nav.scss

```
@@ -0,0 +1,53 @@
+.nav {
+  display: flex;
+  flex-shrink: 1;
+
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item {
+    display: flex;
+    height: $size-header;
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    display: flex;
+    white-space: nowrap;
+    height: $size-header;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $main-color;
+
+    transition: all 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```